server/plugin/plugin.go

```diff
@@ -56,6 +56,8 @@ const (
 	dailySummary                 = "_dailySummary"
 
 	chimeraGitHubAppIdentifier = "plugin-github"
+
+	invalidTokenError = "401 Bad credentials" //#nosec G101 -- False positive
```

Comment:
> Are we sure this error only occurs in case of invalid tokens and in permission issues?
I am not sure of this, but this is the only error we received in our case of handling revoked tokens. When we receive this error, we disconnect the GitHub user and send a bot DM.
Took a reference from the GitLab plugin.

Reply:
> Also, why are we using nosec here?

it considers it as ` G101: Potential hardcoded credentials (gosec)`